install/kubernetes/cilium/values.yaml

```diff
@@ -418,6 +418,14 @@ daemon:
 # program. When not specified, probing will automatically detect devices.
 # devices: ""
 
+# -- Enables experimental support for the detection of new and removed datapath
+# devices. On device change the eBPF datapath, IPsec and BandwidthManager
+# functionality is reloaded to add and remove the device specific eBPF programs.
```

Comment:
```suggestion
# functionalities are reloaded to add and remove the device-specific eBPF programs.
```

Reply:
Thanks. Rewrote the comment completely since the IPsec and BandwidthManager support is TBD, and the wildcards are actually used now to filter the devices.